apartment/api/serializers.py

```diff
@@ -0,0 +1,146 @@
+    sanitation = serializers.CharField(source="project_sanitation")
+    zoning_info = serializers.CharField(source="project_zoning_info")
+    zoning_status = serializers.CharField(source="project_zoning_status")
+    building_type = serializers.CharField(source="project_building_type")
+    description = serializers.CharField(source="project_description")
+    accessibility = serializers.CharField(source="project_accessibility")
+    smoke_free = serializers.CharField(source="project_smoke_free")
+    publication_start_time = serializers.DateTimeField(
+        source="project_publication_start_time"
+    )
+    publication_end_time = serializers.DateTimeField(
+        source="project_publication_end_time"
+    )
+    premarketing_start_time = serializers.DateTimeField(
+        source="project_premarketing_start_time"
+    )
+    premarketing_end_time = serializers.DateTimeField(
+        source="project_premarketing_end_time"
+    )
+    application_start_time = serializers.DateTimeField(
+        source="project_application_start_time"
+    )
+    application_end_time = serializers.DateTimeField(
+        source="project_application_end_time"
+    )
+    material_choice_dl = serializers.CharField(source="project_material_choice_dl")
+    shareholder_meeting_date = serializers.DateTimeField(
+        source="project_shareholder_meeting_date"
+    )
+    estimated_completion = serializers.CharField(source="project_estimated_completion")
+    estimated_completion_date = serializers.DateTimeField(
+        source="project_estimated_completion_date"
+    )
+    completion_date = serializers.DateTimeField(source="project_completion_date")
+    posession_transfer_date = serializers.DateTimeField(
+        source="project_posession_transfer_date"
+    )
+    attachment_urls = serializers.ListField(source="project_attachment_urls")
+    main_image_url = serializers.CharField(source="project_main_image_url")
+    image_urls = serializers.ListField(source="project_image_urls")
+    virtual_presentation_url = serializers.CharField(
+        source="project_virtual_presentation_url"
+    )
+    acc_salesperson = serializers.CharField(source="project_acc_salesperson")
+    acc_financeofficer = serializers.CharField(source="project_acc_financeofficer")
+    project_manager = serializers.CharField(source="project_project_manager")
+    constructor = serializers.CharField(source="project_constructor")
+    housing_manager = serializers.CharField(source="project_housing_manager")
+    estate_agent = serializers.CharField(source="project_estate_agent")
+    estate_agent_email = serializers.CharField(source="project_estate_agent_email")
+    estate_agent_phone = serializers.CharField(source="project_estate_agent_phone")
+    coordinate_lat = serializers.FloatField(source="project_coordinate_lat")
+    coordinate_lon = serializers.FloatField(source="project_coordinate_lon")
+
+    class Meta:
+        fields = (
```

Comment:
Is this list needed in this case? I would imagine that all fields that are specified in the serializer are to be included as well, and since this is not a model serializer there should not be a risk of rogue fields being included.
From what I remember, one does not have to have all fields listed and the Meta class could actually be remove completely in this case.